debian/rules

```diff
@@ -42,6 +42,7 @@ override_dh_auto_install:
 	install -D plugins/hypervisors/kvm/target/cloud-plugin-hypervisor-kvm-$(VERSION).jar $(DESTDIR)/usr/share/$(PACKAGE)-agent/lib/
 	install -D plugins/hypervisors/kvm/target/dependencies/* $(DESTDIR)/usr/share/$(PACKAGE)-agent/lib/
 	install -D plugins/storage/volume/storpool/target/cloud-plugin-storage-volume-storpool-$(VERSION).jar $(DESTDIR)/usr/share/$(PACKAGE)-agent/lib/
+	install -D plugins/storage/volume/storpool/target/cloud-plugin-storage-volume-linstor-$(VERSION).jar $(DESTDIR)/usr/share/$(PACKAGE)-agent/lib/
```

Comment:
@rp- , during the packaging phase the following is reported:
```
install -D plugins/storage/volume/storpool/target/cloud-plugin-storage-volume-linstor-4.18.0.0-shapeblue5122.jar "debian/tmp"/usr/share/cloudstack-agent/lib/
install: cannot stat 'plugins/storage/volume/storpool/target/cloud-plugin-storage-volume-linstor-4.18.0.0-shapeblue5122.jar': No such file or directory
debian/rules:26: recipe for target 'override_dh_auto_install' failed
make[1]: *** [override_dh_auto_install] Error 1
make[1]: Leaving directory '/jenkins/workspace/acs-deb-pkg-builder'
debian/rules:9: recipe for target 'binary' failed
make: *** [binary] Error 2
dpkg-buildpackage: error: debian/rules binary gave error exit status 2
Build step 'Execute shell' marked build as failure
```
I think you made a typo here:
```suggestion
	install -D plugins/storage/volume/linstor/target/cloud-plugin-storage-volume-linstor-$(VERSION).jar $(DESTDIR)/usr/share/$(PACKAGE)-agent/lib/
```
???

Reply:
yep correct, thanks for catching